parcels/plotting.py

```diff
@@ -169,7 +169,10 @@ def plotfield(field, show_time=None, domain=None, depth_level=0, projection=None
             data[1] = d
 
         spd = data[0] ** 2 + data[1] ** 2
-        speed = np.sqrt(spd, where=spd > 0)
+        try:
+            speed = np.sqrt(spd, where=spd > 0, out=np.zeros(spd.shape))
```

Comment:
@erikvansebille, did you see this strange error?
It was breaking on py3 linux setup.

Anyway, I would suggest to only keep the except part of the fix, which is valid for everyone.

By the way, old code was not filling values where the `where` condition was not met (see [here](https://docs.scipy.org/doc/numpy/reference/generated/numpy.sqrt.html))
So there was some uninitialized values, out argument is important here)

Reply:
Yes, I agree to keep only the `speed = np.where(spd > 0, np.sqrt(spd), 0)` statement. Can you change that?